.github/workflows/rust.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Build
         run: cargo build
       - name: Generate
-        run: cargo run -- manifests > manifests/crd.yml
+        run: cargo run -- manifests > config/crd/bases/kubecfg.dev_appinstances.yaml
```

Comment:
in another PR you can modify the rust code so that it writes directy in `config/crd/<filename>` instead of emitting to stdout. This allows us to emit multiple CRDs and have them live nicely in multiple files

Reply:
I'll give it a shot :eyes: